data/strategies/StrategyMakerDelegate.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "strategy",
+    "name": "Maker Delegate Strategy",
+    "description": "Locks {{token}} in a [MakerDAO](https://cdp.makerdao.com/) vault and mints DAI. This newly minted DAI is then deposited into the v2 DAI yVault to earn yield.",
```

Comment:
> Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?

Reply:
This is what I have in Vaults at Yearn.

"Locks YFI at MakerDAO via YFI-A to mint DAI. The newly minted DAI is then deposited into the v2 DAI yVault to earn yield." I would like to show which type of vault each strategy uses if possible. Not sure if we can pull this data from the contract, but I don't believe it is in the contract. So I think if we want to add vault type, these cannot use variables and need to be individual descriptions for each delegated strategy.

Maybe we can use `c` as a variable {{c}}/100 to show the lowest collateralization ratio we consider safe?